app/assets/stylesheets/_settings.scss

```diff
@@ -382,7 +383,7 @@ $mediaobject-image-width-stacked: 100%;
 
 $menu-margin: 0;
 $menu-margin-nested: 1rem;
-$menu-item-padding: 0.7rem 1rem;
+$menu-item-padding: 0.7rem 1.3rem;
```

Comment:
this caused the problem with 'Donate' button (anything bigger than 1024px wide), but it was already an issue on certain large screen sizes (and problem may re-appear if we change link names in the menu), so @abyczkowska needs figure out a design solution for this.